### ISSUE TITLE
BrowserBox Tor Hidden Service

### ISSUE THREAD

**Issue by FlamestoN** [FlamestoN/BrowserBox#2]
# Set Up a Tor Hidden Service

Before starting, [fork](../fork) or [generate](../generate) this repo to your account (not for orgs).

Then ensure that:

1. [Issues](settings#issue-feature) are switched on, and 
2. [Actions](actions) are enabled.

To begin the action to create your BrowserBox Tor Hidden Service, click **Submit New Issue**. 

Your `.onion` login link will appear in the comments after a few minutes.

*Note: This service activates only when you first create or reopen the issue. It won't consume your GitHub Actions minutes unnecessarily. Only you will be able to run this job by opening this issue, nobody else's issue will be able to trigger it. Edit `userjob.yaml` to change this.*

---

*BrowserBox by [DOSYAGO](https://dosyago.com)*

**Support Tor**: [Donate to Tor](https://donate.torproject.org)
